pkg/reconciler/autoscaling/kpa/kpa.go

```diff
@@ -258,8 +268,8 @@ func reportMetrics(pa *autoscalingv1alpha1.PodAutoscaler, pc podCounts) {
 //	| -1   | >= min | 0     | active     | inactive   | <-- this case technically is impossible.
 //	| -1   | >= min | >0    | activating | active     |
 //	| -1   | >= min | >0    | active     | active     |
-func computeActiveCondition(ctx context.Context, pa *autoscalingv1alpha1.PodAutoscaler, pc podCounts) {
-	minReady := activeThreshold(ctx, pa)
+func computeActiveCondition(ctx context.Context, pa *autoscalingv1alpha1.PodAutoscaler, spa *autoscalingv1.StagePodAutoscaler, pc podCounts) {
+	minReady := activeThreshold(ctx, pa, spa)
```

Comment:
Let's say `minScale` is 4, `maxScale` is 10 and the `currentScale` is 8; when we set `stagleMinScale` to 2 we need to move 2/8=25% traffic to the new revision, how do we control the traffic routing with the revisions ?